### ISSUE TITLE
App stuck in training

### ISSUE THREAD

**Issue by HeyArian** [wit-ai/wit#2360]
**Do you want to request a *feature*, report a *bug*, or ask a *question* about wit?**
Bug
**What is the current behavior?**
App is stuck in "Training ongoing" for almost a day.
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Open the app. Wait until the app is done training. Training never stops (even though it normally takes less than a fifteen minutes, half an hour at most).
**What is the expected behavior?**
App is expected to finish training in some time.
**If applicable, what is the App ID where you are experiencing this issue? If you do not provide this, we cannot help.**
790110095172328

Thanks in advance for helping :)

**Comment by AlexisNichel** [comment]
same here plz helpme, app id = 1856185531061946

**Comment by wangzhenict** [comment]
> same here plz helpme, app id = 1856185531061946

unblocked it from another ticket.

**Comment by wangzhenict** [comment]
Hi @HeyArian,
I'm working on unblocking it, will update later.

**Comment by wangzhenict** [comment]
Hi @HeyArian 
update:
our logs showed that your training is successful, we are still investigating why the UI didn't show it properly.

Thanks for your patience!

**Comment by jqol** [comment]
Same here plz help me, https://wit.ai/apps/531409674821311

**Comment by wangzhenict** [comment]
> https://wit.ai/apps/531409674821311

Unblocked.

**Comment by sophiexr** [comment]
same for app 572046700564927

thanks

**Comment by Ananyag02** [comment]
Requesting you to kindly let me know if I can extend the utterance for training the dataset or let me know if there is any paid version.

**Comment by doggeral** [comment]
@HeyArian your app has been fixed!
